app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -117,6 +127,22 @@ export default {
   watch: {
     "session.signedIn": function() {
       this.loadStakingTxs()
+    },
+    lastHeader: {
+      immediate: true,
+      handler(newHeader) {
+        const waitTenBlocks = Number(newHeader.height) % 20 === 0
```

Comment:
```suggestion
        const waitTenBlocks = Number(newHeader.height) % 10 === 0
```
why sometimes 10 and sometimes 20?

Reply:
10 is for data that's displayed on the balance header